src/components/screens/index.tsx

```diff
@@ -47,13 +47,5 @@ export default function WalletConnect({
     analytics.start(userId, walletInfo);
   }, []);
 
-  return (
-    <Home
-      initialMode={initialMode}
-      setUri={setUri}
-      accounts={accounts}
-      initialURI={uri}
-      {...rest}
-    />
-  );
+  return <Home initialMode={initialMode} setUri={setUri} initialURI={uri} {...rest} />;
```

Comment:
I know it was already done like this before, but I'm never confident to spread at the end as you allow override of props set before. Is it intentional?